src/App.test.tsx

```diff
@@ -2,10 +2,8 @@ import React from 'react'
 import ReactDOM from 'react-dom'
 import App from './App'
 
-import election from './data/election.json'
-
 it('renders without crashing', () => {
   const div = document.createElement('div')
-  ReactDOM.render(<App election={election} />, div)
+  ReactDOM.render(<App election={undefined} />, div)
```

Comment:
is this the right approach? My sense is that making this nullable introduces the chance for problems. Feels like either `App` should have no parameter and have the config upload capability within it, or `App` should have a parameter that is non-nullable and the config upload capability should be outside of it.

What do you think?

Reply:
Yes, I agree. I was considering making a sister to `App` component which is the `Config` component which is loaded when there is no election value in localStorage. Read the description in this issue and add your comments:

- Use localStorage for persisting loaded election config data #24